### DISCUSSION TITLE
Mingit and ssh-keygen

### DISCUSSION THREAD

**Discussion by hummeleBop** [git-for-windows/git#3033] [Ideas]
Could I ask for the ssh-keygen/ssh-keyscan availability in the MinGit bundle ?
For security reasons, I think this tool is fair enough to be part of the MinGit bundle.

**Comment by dscho** [comment]
You will have to change [this line](https://github.com/git-for-windows/build-extra/blob/f55e87476d191082ffa12eacdfcaf22a84f13a63/make-file-list.sh#L260).

My main concern is the size of the MinGit package. Therefore, it will be necessary to build it via `sdk cd build-extra && ./mingit/release.sh 0-test` in [the Git for Windows SDK](https://gitforwindows.org/#download-sdk), both with and without that change, then include the sizes of the two `.zip` files in the commit message, by way of making a case that the change does not "cost" too much.

**Comment by dscho** [comment]
@hummeleBop if you are still interested, please feel free to follow the advice given above and open a PR.
